test/string_extension_test.dart

```diff
@@ -330,4 +330,57 @@ void main() {
       expect(result, equals(text));
     });
   });
+
+  group('msisdnSanitizer tests', () {
```

Comment:
how about the case if phone number is:
- (+84) 366 6769 439
- +84-366-769-439
- +84-3667-69439

Reply:
added test cases